src/test/java/org/junitpioneer/testkit/assertion/ExceptionAssert.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.testkit.assertion;
+
+/**
+ * Used to assert exceptions thrown by failed tests/containers.
+ */
+public interface ExceptionAssert {
```

Comment:
could we actually use https://github.com/joel-costigliola/assertj-core/blob/master/src/main/java/org/assertj/core/api/AbstractThrowableAssert.java instead of that assert - it does have already a lot of functionality and we would stick to the default

Reply:
We *could* use it but it would look kind of broken:
```
assertThat(results)
    .hasSingleTest()
    .thatFailed()
    .withException()
    .hasMessageContainingAll(/* ... */);
```
Then again, this interface does not really do anything besides adding the word `that` in front of the methods in `AbstractThrowableAssert`

Reply:
but it would also provide additional methods, like message containing with varargs, etc.

Reply:
No need to reinvent the wheel, you are right.